smrf/distribute/solar.py

```diff
@@ -243,6 +243,7 @@ def initialize(self, topo, data):
         """
 
         self._logger.debug('Initializing distribute.solar')
+        self.date_time = date_time
         # Solar has no stations. Relies on Cloud factor
         self.stations = None
         self._initialize(topo, data.metadata)
```

Comment:
Looks like 'initialize' should be in the `image_data.image_data` and calls `self._initialize` then all inheriting classes call `super()` and add their customization

Reply:
Makes sense, might be a bit of scope creep on this PR though

Reply:
#185